source/_docs/settings-php.md

```diff
@@ -108,123 +108,32 @@ if (defined('PANTHEON_ENVIRONMENT')) {
-        // Google Analytics.
-        $conf['googleanalytics_account'] = 'UA-XXXXXXXX-Z';
-      }
-    }
-
-The Drupal 8 [configuration override system](https://www.drupal.org/node/1928898) uses the global `$config` variable:
-
-    // All Pantheon Environments.
-    if (defined('PANTHEON_ENVIRONMENT')) {
-      // Drupal caching in test and live environments.
-      if (in_array(PANTHEON_ENVIRONMENT, array('test', 'live'))) {
-        // Expiration of cached pages - 15 minutes.
-        $config['system.performance']['cache']['page']['max_age'] = 900;
-        // Aggregate and compress CSS files in Drupal - on.
-        $config['system.performance']['css']['preprocess'] = true;
-        // Aggregate JavaScript files in Drupal - on.
-        $config['system.performance']['js']['preprocess'] = true;
-        // Google Analytics.
-        $config['google_analytics.settings']['account'] = 'UA-xxxxxxx-xx';
-      }
-      // Drupal caching on dev environment, and all multidevs
-      else {
-        // Expiration of cached pages - none.
-        $config['system.performance']['cache']['page']['max_age'] = 0;
-        // Aggregate and compress CSS files in Drupal - off.
-        $config['system.performance']['css']['preprocess'] = false;
-        // Aggregate JavaScript files in Drupal - off.
-        $config['system.performance']['js']['preprocess'] = false;
-      }
-    }
-
-#### How can I read the Pantheon environmental configuration, like database credentials?
-
-See [Reading the Pantheon Environment Configuration](/docs/read-environment-config).
-
-#### Why does Drupal report that `settings.php` is not protected? I can't change the permissions on `settings.php`.
-
-If you do not have a `settings.php` file in your codebase, you'll see the following message on `/admin/reports/status`:
-
+Yes, but only if at least one other file (e.g. `settings.php`) is present within the `sites/default` directory. Otherwise, the existing symlink to `sites/default/files` will be invalid.
+
+#### How can I write logic based on the Pantheon server environment?
+
+Depending on your use case, there are three possibilities:
+
+ - For web only actions, like redirects, check for the existence of `$_SERVER['PANTHEON_ENVIRONMENT']`. If it exists, it will contain a string with the current environment (Dev, Test, or Live. See our [redirects](/docs/redirects/#redirect-to-www) guide for examples.
+
+ - For actions that should take place on every environment, such as Redis caching, use the constant `PANTHEON_ENVIRONMENT`. Again, it will contain Dev, Test, or Live. See our [Redis](/docs/redis) guide for examples for [Drupal 8](https://pantheon.io/docs/redis#drupal-8-sites) or [Drupal 7](/docs/redis#drupal-7-sites)
+
+ - For Actions that require access to protected services like Redis or the site database, you can use the `$_ENV` superglobal. Please review our guide on [Reading Pantheon Environment Configuration](/docs/read-environment-config/) for more information.
```

Comment:
The linked doc doesn't show an example snippet where this is used (we should still keep it). But maybe we can reuse the same redis links from bullet 2, the snippets use both methods `PANTHEON_ENVIRONMENT` and `$_ENV`